src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/entities/BailCaseFieldDefinition.java

```diff
@@ -255,6 +255,8 @@ public enum BailCaseFieldDefinition {
         "detentionFacility", new TypeReference<String>(){}),
     UPLOAD_BAIL_SUMMARY_DOCS(
         "uploadBailSummaryDocs", new TypeReference<List<IdValue<DocumentWithDescription>>>(){}),
+    UPLOAD_B1_FORM_DOCS(
```

Comment:
Is it not rebased on latest branch? These were my changes which went in this morning, Can you please check.

Reply:
I have rebased